frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -83,6 +83,6 @@ function onClick(store: Store) {
   return () => {
     let dataset = createDataset(store)
     let jsonDataset = JSON.stringify(dataset, null, 4)
-    download(jsonDataset, 'json', `bayesimpact-dataset-${dataset.dataSources.join('_')}.json`)
+    download(jsonDataset, 'json', `bayesimpact-dataset-${dataset.dataSources}.json`)
```

Comment:
Should we sanitize this maybe?

Reply:
what would you like it to look like?

Reply:
![image](https://user-images.githubusercontent.com/11825994/37051270-cdedaf82-212a-11e8-9930-227a5811c517.png)
updated download analysis link too